content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -5,33 +5,54 @@ DVCLive allows you to add experiment tracking capabilities to your
 
 ## Usage
 
-Pass the
+<p align='center'>
+  <a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-PyTorch-Lightning.ipynb">
+    <img src="https://colab.research.google.com/assets/colab-badge.svg" />
+  </a>
+</p>
+
+If you pass the
 [`DVCLiveLogger`](https://github.com/iterative/dvclive/blob/main/src/dvclive/lightning.py)
 to your
-[`Trainer`](https://pytorch-lightning.readthedocs.io/en/latest/common/trainer.html):
+[`Trainer`](https://lightning.ai/docs/pytorch/stable/common/trainer.html),
+DVCLive will automatically log the
+[metrics](https://lightning.ai/docs/pytorch/stable/visualize/logging_basic.html#track-metrics)
+and
+[parameters](https://lightning.ai/docs/pytorch/stable/common/lightning_module.html#save-hyperparameters)
+tracked in your
+[`LightningModule`](https://pytorch-lightning.readthedocs.io/en/2.0.1/common/lightning_module.html).
 
 ```python
+import lightning.pytorch as pl
 from dvclive.lightning import DVCLiveLogger
 
 ...
+class LitModule(pl.LightningModule):
+    def __init__(self, layer_1_dim=128, learning_rate=1e-2):
+        super().__init__()
+        # layer_1_dim and learning_rate will be logged by DVCLive
+        self.save_hyperparameters()
+
+    def training_step(self, batch, batch_idx):
+        metric = ...
+        # See Output Format bellow
+        self.log("train_metric", metric, on_step=False, on_epoch=True)
+
 dvclive_logger = DVCLiveLogger()
 
-trainer = Trainer(logger=dvclive_logger)
+model = LitModule()
+trainer = pl.Trainer(logger=dvclive_logger)
 trainer.fit(model)
 ```
 
-Each metric will be logged to:
-
-```py
-{Live.plots_dir}/metrics/{split}/{iter_type}/{metric}.tsv
-```
+<admon type="info">
 
-Where:
+By default, PyTorch Lightning creates a directory to store checkpoints using the
+logger's name (`DVCLiveLogger`). You can change the checkpoint path or disable
+checkpointing at all as described in the
+[PyTorch Lightning documentation](https://lightning.ai/docs/pytorch/stable/common/checkpointing_basic.html#disable-checkpointing)
```

Comment:
We have this admon and then a snippet in the examples about how to checkpoint. Should we be more opinionated and suggest one workflow here, or at least connect between this admon and the example below somehow?

Reply:
I think we could drop the admon and leave the snippet as reference for working with checkpoints

Reply:
Let's make it a follow-up issue. Don't want to block this PR.